src/components/buttonPlay/buttonPlay.js

```diff
@@ -0,0 +1,7 @@
+export const buttonPlay = (label, onClick) => {
+  const buttonTemplateHTML = `<button class='gameButton gameButton_play'>${label}</button>`;
+  if (onClick) {
+    buttonPlay.addEventListener('click', onClick);
+  }
+  document.getElementById('swquiz-app').innerHTML = buttonTemplateHTML;
+};
```

Comment:
```suggestion
  }
  return buttonTemplateHTML;
};
```
widziałam komentarz do komponentu drugiego przycisku, żeby nie wstawiać bezpośrednio na stronę. Czy ja też powinnam go tylko zwrócić?